docs/src/components/Demo.tsx

```diff
@@ -50,16 +42,17 @@ export const Demo = ({
           <View overflow="auto" padding="5px">
             {children}
           </View>
+          <Divider
+            label="Props"
+            margin="20px 0 0"
+            border={`2px solid ${tokens.colors.brand.primary[80]}`}
```

Comment:
This is a nit, but IMO, I think we should leave this as a gray or neutral color (border.color.secondary maybe?); I think the brand colors draw a little too much attention. Even removing the label would be fine since this is a pretty "busy" visual area
<img width="300" alt="Screen Shot 2022-08-09 at 3 29 52 PM" src="https://user-images.githubusercontent.com/376920/183745527-b8d758a1-34c4-4695-8a23-d0435c847c4d.png">
.

Reply:
or if keeping the label, whatever gray gives enough contrast against the background, might have to manually check what that is since we use an opacity to lighten the Divider. Maybe something else for us to update if we add tokens for customizing the label color and divider color; the opacity makes it a little harder to theme just how you want it.

Reply:
I agree with @hbuchel  about leaving it as a neutral gray color and removing the label